src/ra_server.erl

```diff
@@ -297,13 +300,17 @@ init(#{id := Id,
         case ra_log:recover_snapshot(Log0) of
             undefined ->
                 InitialMachineState = ra_machine:init(Machine, Name),
-                {0, make_cluster(Id, InitialNodes),
+                Clu = make_cluster(Id, InitialNodes,
```

Comment:
is it possible to set initial nodes which are non voters? if not we shouldn't need to take the initial voter status from config here.

Reply:
This value is used to add the node itself during a cluster join, L2262. Since there is a top-level voter_status, seeding the one within cluster is optional, but I chose to keep it consistent. LMK if you prefer it pure.